.github/workflows/wrangler.toml

```diff
@@ -0,0 +1,11 @@
+name = "cdn-worker"
+type = "javascript"
+zone_id = ""
+account_id = "78e1d5140b47fc9dab18dc8b25351b7a"
+compatibility_date = "2021-10-08"
+route = ""
+workers_dev = true
+kv_namespaces = [
+  { binding = "RTV", id = "f33b97f2e1434004975c0d8553f70488", preview_id = "192df3dff1b345ce8e7c794329bb7ccd" },
+  { binding = "AMP_EXP", id = "b03e5db05f3e4d068508da267e248949", preview_id = "69318ded67c0477b85d73ec801b47543" }
+]
```

Comment:
Will the IDs in this file have to be updated from time to time? Any way to avoid hard-coding them here and ingesting them via a secure environment variable?

Also, can we comment these lines (assuming `.toml` files support it) with notes about what the IDs refer to?